src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -26,14 +26,9 @@ export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
     if (meta !== undefined && meta !== null) {
       return (
         <div className="menuButtonsMetaInfoButtonBox">
-          <div className="menuButtonsMetaInfoButtonLabel">
-            <span className="menuButtonsMetaInfoButtonLabelOverflow">
-              {_formatLabel(meta)}
-            </span>
-          </div>
           <ButtonWithPanel
             className="menuButtonsMetaInfoButtonButton"
-            label="&nbsp;"
+            label={_formatLabel(meta)}
```

Comment:
I think we should default something in case it's empty, otherwise we won't be able to click. Maybe 'Profile information' ?
=> `label={_formatLabel(meta) || 'Profile information'}`

Reply:
Yeah, we should do it. This thought didn't came to my mind earlier though :(